ocaml/xapi/repository.ml

```diff
@@ -288,9 +288,7 @@ let get_hosts_updates ~__context =
       )
       hosts
   in
-  with_pool_repositories (fun () ->
-      Helpers.run_in_parallel ~funs ~capacity:capacity_in_parallel
-  )
+  Helpers.run_in_parallel ~funs ~capacity:capacity_in_parallel
```

Comment:
I think we still need to block checking updates when the updates is being applied.

Reply:
I recovered the mutex lock. Only enable the handler `/repository`.